src/main/webapp/rolesManagement/RolesManagement.js

```diff
@@ -0,0 +1,182 @@
+import { Component } from 'react';
+import { h, div, span } from 'react-hyperscript-helpers';
+import { Table } from "../components/Table";
+import { RoleManagementEdit } from "../components/RoleManagementEdit";
+import { User } from "../util/ajax";
+import { spinnerService } from "../util/spinner-service";
+import { Spinner } from '../components/Spinner';
+import { TablePaginator } from "../components/TablePaginator";
+
+const tableHeaders =
+  [
+    { name: 'User Name', value: 'userName' },
+    { name: 'Display Name', value: 'displayName' },
+    { name: 'Email Address', value: 'emailAddress' },
+    { name: 'Roles', value: 'roles' },
+  ];
```

Comment:
This is the right idea ... each one of these should also have a function for `dataFormat` so that Table doesn't have to know about it. It would take some refactoring around how we're using Table in other places too.